tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/errors/FunctionNotFoundError.java

```diff
@@ -0,0 +1,35 @@
+package com.google.cloud.bigquery.utils.queryfixer.errors;
+
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.utils.queryfixer.entity.Position;
+
+import lombok.Getter;
+
+/** A class to represent the "Function Not Found" errors. The errors look like one of the two forms: (1) Function not
+ * found: [function] at [x:y] or (2) Function not found: [function]; Did you mean [suggestion]? at [x:y], where
+ * [function] is the incorrect function, [suggestion] is one of the correct function candidates, and [x:y] is the row
+ * and column numbers. */
+@Getter
+public class FunctionNotFoundError extends BigQuerySemanticError {
+
+  private final String incorrectFunctionName;
+  private final String suggestion;
+
+
+  public FunctionNotFoundError(String incorrectFunctionName, Position errPos, String suggestion,
```

Comment:
nit: functionName instead of 'incorrectFunctionName'

Reply:
nit: errPos -> position,

Reply:
Done